### DISCUSSION TITLE
Ecosia

### DISCUSSION THREAD

**Discussion by isabelroses** [catppuccin/catppuccin#2020] [Port Requests]
### Will you make the port?

- [X] Yes, I'm willing to make the port

### How can the suggested port be themed?

Using stylus. Through `:root` variables.

Completed repo:
https://github.com/isabelroses/Ecosia

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by sgoudham** [comment]
Closing since transferred to #2021
